styles.css

```
@@ -1523,22 +1523,39 @@ pre,
+    text-decoration-thickness: 0.08em;
+    text-underline-offset: 0.18em;
+    transition: color 0.15s ease, text-decoration-color 0.15s ease;
+    width: max-content;
 }
 
 .agents-content-item:hover,
 .agents-content-item:focus-visible {
     color: var(--accent);
-    text-decoration: underline;
+    outline: none;
+    text-decoration-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)